### PR TITLE
OV7251 updates for libcamera compatibility.

### PR DESCRIPTION
**Pull request by 6by9**
7 of the patches are from the linux-media mailing list. Minor changes were requested, and I've got a fixup on one of them, but they otherwise seem reasonable. I ought to submit the rest of my patches upstream.

Support requested by https://github.com/raspberrypi/libcamera-apps/issues/235

### CODE REVIEW

**Comment by 6by9** [comment]
@naushir and @davidplowman to review.

File: drivers/media/i2c/ov7251.c [hunk 1]
```diff
@@ -37,6 +37,8 @@
 #define OV7251_AEC_EXPO_2		0x3502
```
Comment (pelwell) [inline drivers/media/i2c/ov7251.c:37]: There's a typo in the commit message - `depenedent`.

**Review by pelwell — COMMENTED** [review]
Apart from the typo in the commit message for https://github.com/raspberrypi/linux/pull/4897/commits/30a3d59438b1533bef77f37c88956bc1f1498efa I couldn't spot any issues (it forward-ports cleanly).

**Comment by pelwell** [comment]
Merged offline with the typo fixed.

**Pull request closed without merging**